docker/ACEseqDockerContainer/roddy.sh

```diff
@@ -92,13 +93,15 @@ call="${roddyBinary} ${mode} ${configurationIdentifier}@copyNumberEstimation ${p
 absoluteCall="[[ ! -d ${workspace}/${pid} ]] && mkdir ${workspace}/${pid}; $prepareAdditionalConfigCall ;$call; echo \"Wait for Roddy to finish\"; "'while [[ 2 -lt $(qstat | wc -l ) ]]; do echo $(expr $(qstat | wc -l) - 2 )\" jobs are still in the list\"; sleep 120; done;'" echo \"done\"; ec=$?"
  
 docker run \
-		-v ${inputBamCtrlLcl}:${inputBamCtrl} -v ${inputBamCtrlLcl}.bai:${inputBamCtrl}.bai \
-		-v ${inputBamTumorLcl}:${inputBamTumor} -v ${inputBamTumorLcl}.bai:${inputBamTumor}.bai \
-		-v ${workspaceLcl}:${workspace} \
-		-v ${referenceGenomePath}:${referenceGenomePath} \
-		-v "${referenceFilesPath}:${referenceFilesPath}" \
-		-v "${configurationFolderLcl}:${configurationFolder}" \
+		-v "${inputBamCtrlLcl}:${inputBamCtrl}:ro" -v "${inputBamCtrlLcl}.bai:${inputBamCtrl}.bai:ro" \
+		-v "${inputBamTumorLcl}:${inputBamTumor}:ro" -v "${inputBamTumorLcl}.bai:${inputBamTumor}.bai:ro" \
+		-v "${workspaceLcl}:${workspace}" \
+		-v "${referenceGenomePath}:${referenceGenomePath}:ro" \
+		-v "${referenceFilesPath}:${referenceFilesPath}:ro" \
+		-v "${configurationFolderLcl}:${configurationFolder}:ro" \
+		${svFileMount} \
 		--rm \
+		--shm-size=1G \
```

Comment:
for what is the shared mem option?

Reply:
R uses shared memory to send/receive data to/from system commands. If the memory is too small, launching system commands fails, because the output of the command cannot be buffered. The default is 64 Mb, I believe.